docs/requirements.txt

```diff
@@ -5,6 +5,8 @@ breathe==4.16.0
 sphinxcontrib.programoutput
 #sphinxcontrib-napoleon>=0.7
 pygments
+Jinja2<3.0
```

Comment:
Is this future proof? Do we need a version update in the future?

Reply:
I do not know this is fu..ing python^^. I tested it on my local machine nd if I used something newer I got version conflicts with other libs.